lib/Models/CatalogItem.js

```diff
@@ -320,13 +325,81 @@ var CatalogItem = function(terria) {
         }
     });
 
+    /**
+     * Gets the CatalogItems current time.
+     *
+     * This property is an observable version of clock.currentTime, they will always have the same value.
+     *
+     * This property is effectively an interface adapter for clock.definitionChanged which changes the structure from
+     * an Event when the current time changes to a knockout property which can be observed.
+     *
+     * @member {JulianDate} The CatalogItems current time.
```

Comment:
The value in the position of `The` here is meant to be the name of the property. The reference doc currently looks like this as a result:
![image](https://user-images.githubusercontent.com/924374/36456853-29146840-16fb-11e8-86db-0a04fa746d85.png)

Reply:
Actually that image is for the property below, not this one, but same problem.